docker-compose.yml

```diff
@@ -25,7 +25,7 @@ services:
         restart: "on-failure:10"
 
     postgres:
-        image: codefordc2/housing-insights-postgres
+        image: mdillon/postgis
```

Comment:
This was something owen changed way back when and I just never changed it back.  I think at one point we were considering taking advantage of some `postgis` capability, but maybe we never went that route.

I'll change it back to the original one since there doesn't seem to be any reason to use the `postgis` image right now.